masschallenge/django-accelerator
# [AC-9548] 2. Add Team Nav item to the revised Enterprise edit view

### [AC-9548](https://masschallenge.atlassian.net/browse/AC-9548):
- Added subnav item migration

**How to test:**
- On your local environment, run `git checkout AC-9548; git pull`.
- Run `make migrate` on accelerate
- As a logged-in user, access http://localhost:8181/mycompanies
- Or Access https://test-cam.masschallenge.org/mycompanies/
- Ensure that the definition of done according to [AC-9548](https://masschallenge.atlassian.net/browse/AC-9548) is satisfied.

**[Sibling PR](https://github.com/masschallenge/accelerate/pull/3312)**
